lib/commands/validate-example.ts

```diff
@@ -33,9 +34,11 @@ export async function handler(argv: yargs.Arguments): Promise<void> {
       pretty: argv.p
     }
     if (specPath.match(/.*composite.*/gi) !== null) {
-      await validate.validateExamplesInCompositeSpec(specPath, vOptions)
+      const result = await validate.validateExamplesInCompositeSpec(specPath, vOptions)
+      return iterator.flat(result).some(() => true) ? 1 : 0
     } else {
-      await validate.validateExamples(specPath, operationIds, vOptions)
+      const result = await validate.validateExamples(specPath, operationIds, vOptions)
+      return iterator.some(result, () => true) ? 1 : 0
```

Comment:
is result here an array of errors and empty if no errors exist? if so, why not return `result.length > 0 ? 1 : 0`